tests/data/mock_manifests/InvalidFilenameAndEntityIDManifest.csv

```diff
@@ -2,5 +2,5 @@ Component,Filename,entityId
 MockFilename,schematic - main/MockFilenameComponent/txt1.txt,syn61682653
 MockFilename,schematic - main/MockFilenameComponent/txt2.txt,syn61682660
 MockFilename,schematic - main/MockFilenameComponent/txt3.txt,syn61682662
-MockFilename,schematic - main/MockFilenameComponent/txt4.txt,syn6168265
-MockFilename,schematic - main/MockFilenameComponent/txt5.txt,
```

Comment:
So in the original feature request, the two conditions that needed to be covered for validation were
>- if filename/path in a manifest does not exist in the corresponding Synapse dataset folder: Provide error indicating which row(s) do not have valid filename/path(s)

which was addressed by line 6 in the manifest here
and
>- if filename/path exists but is associated with a different entity ID than the one provided in the manifest: Provide error indicating which row(s) have filename/path(s) which do not match the entity ID provided in the manifest

which was addressed by line 5 in the manifest here

to test the entityID validation you should add new rows or a separate manifest/dataset combination

Reply:
Ah, my bad I misinterpreted that manifest CSV file and test. I will revert these changes and then circle back to this on Monday.